templates/traefik/config.yml

```diff
@@ -13,6 +13,6 @@ description: |
   Traefik works by using Docker labels which have been assigned to containers to define the routing table. See https://github.com/rawmind0/rancher-traefik#configuration-labels.
 
   NOTE: Rancher health checks need to be enable on a service, otherwise Traefik won't accept the config.
-version: v1
+version: 1.0.0
 category: nhsuk beta load-balancer
 projectURL: https://github.com/nhsuk/nhsuk-rancher-templates
```

Comment:
Ah, this one actually refers to https://github.com/nhsuk/nhsuk-rancher-templates/blob/master/templates/traefik/0/rancher-compose.yml#L3
I'll change that to match the same sem ver scheme